docs/package_templates/autotools_package/all/conanfile.py

```diff
@@ -158,7 +158,7 @@ def generate(self):
             env.define("CC", f"{compile_wrapper} cl -nologo")
             env.define("CXX", f"{compile_wrapper} cl -nologo")
             env.define("LD", "link -nologo")
-            env.define("AR", f"{ar_wrapper} \"lib -nologo\"")
```

Comment:
The `-nologo` option saves a few spammy lines from the build logs, but it has caused the build to fail with a "could not determine ar interface"  in several recipes for me, so it's safer to not include it.